R/hacker_news.r

```diff
@@ -12,15 +28,39 @@ base_url <- "https://hacker-news.firebaseio.com/v0"
 #' get_top_stories(limit = 5, pretty_print = TRUE)
 #' @importFrom jsonlite fromJSON
 get_top_stories <- function(limit = 10, pretty_print = FALSE) {
-    top_stories_ids <- fromJSON(paste(base_url, "/topstories.json", sep=""))
-    top_stories_ids <- top_stories_ids[1:limit]
-    top_stories <- lapply(top_stories_ids, function(id) {
-        story_url <- paste(base_url, "/item/", id, ".json", sep="")
-        fromJSON(story_url)
-    })
-    process_stories(top_stories, pretty_print)
+  top_stories <- lapply(get_item_ids_for_top_stories(), function(id) {
+    story_url <- paste(base_url, "/item/", id, ".json", sep="")
+    fromJSON(story_url)
+  })
+  process_stories(top_stories, pretty_print)
 }
 
+get_new_stories <- function(limit = 10, pretty_print = FALSE) {
+  new_stories <- lapply(get_item_ids_for_new_stories(), function(id) {
+    story_url <- paste(base_url, "/item/", id, ".json", sep="")
+    fromJSON(story_url)
+  })
+  process_stories(top_stories, pretty_print)
```

Comment:
Shouldn't this be `new_stories`.  Same goes for the other calls to `process_stories` where they need to be updated from `top_stories`